packages/datadog_flutter_plugin/lib/src/datadog_configuration.dart

```diff
@@ -179,15 +181,25 @@ class RumConfiguration {
   ///
   /// The sampling rate must be a value between `0.0` and `100.0`. A value of
   /// `0.0` means no RUM events will be sent, `100.0` means all sessions will be
-  /// kept
+  /// sent
   ///
   /// Defaults to `100.0`.
   double sampleRate;
 
+  /// Sets the sampling rate for tracing
+  ///
+  /// The sampling rate must be a value between `0.0` and `100.0`. A value of
+  /// `0.0` means no resources will include APM tracing, `100.0` resource will
+  /// include APM tracing
+  ///
+  /// Defaults to `100.0`.
+  double tracingSamplingRate;
+
   RumConfiguration({
     required this.applicationId,
-    this.sampleRate = 100.0,
-  });
+    double sampleRate = 100.0,
+    this.tracingSamplingRate = 100.0,
+  }) : sampleRate = max(0, min(sampleRate, 100));
```

Comment:
I'm not 100% clear what this constructor does (but that is because I don't know Dart). I suppose it defines sampling values if they are not provided explicitly in the configuration argument. But then why we do `max-min` boundary only for `sampleRate`, but not for `tracingSamplingRate`?

Reply:
No, you got it... this looks like a bug in both the test and the implementation. I was mixing up sample rate and tracingSampleRate, which is exactly why I should take your suggestion above and rename sample rate.